Analysis/source/CalcMulticlassDNN.cxx

```diff
@@ -251,45 +290,14 @@ void FeatureProvider::calculate(Long64_t i) {
   setFeature("tauh_sv_eta", tauH.Eta(), true);
   setFeature("tauh_sv_phi", tauH.Phi(), true);
   setFeature("tauh_sv_e", tauH.E(), true);
+  setFeature("tauh_sv_ez", pow(pow(tauH.Pz(), 2) + pow(tauH.M(), 2), 0.5), true);
```

Comment:
Any clue why for SVfit passing only the p4 is not enough while for all the other variables NN is able to perform on p4 only? Also Ez is a bit unexpected variable, I would say...

Reply:
This was actually a bug on our side. Up to (and including) version "v3", we used the "ez" value by accident but consistently during all trainings. Since we passed eta and phi as well, we don't expect any drawbacks from this. From version "v4" onwards this is changed.